src/com/google/enterprise/adaptor/CustomFormatter.java

```diff
@@ -65,6 +65,24 @@ public CustomFormatter() {
     }
```

Comment:
If we're deprecating useColor, then we should log a message about it here, and update the class javadoc to mention Color and NoColor instead of useColor, or at least in preference to it.